### DISCUSSION TITLE
Release v1.0.14

### DISCUSSION THREAD

**Discussion by moabu** [JanssenProject/jans#4946] [Polls]
https://github.com/JanssenProject/jans/milestone/17

**Comment by manojs1978** [comment]
https://github.com/JanssenProject/jans/issues/5157

**Reply by ossdhaval** [reply]
@manojs1978 Do you see above issue as blocker? It is not clear from your comment above.

**Reply by manojs1978** [reply]
yes it was blocker but resolved now

**Comment by ossdhaval** [comment]
@manojs1978 Can you post consolidated QA status?

**Reply by manojs1978** [reply]
sure @ossdhaval 
will post in some time, currently checking above issue on all platform

**Reply by manojs1978** [reply]
getting config-api service unavailable error in latest nightly build.

https://github.com/JanssenProject/jans/issues/5177
